src/System.Runtime.Numerics/tests/BigInteger/BigIntegerToStringTests.netcoreapp.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Numerics.Tests
+{
+    public partial class ToStringTest
+    {
+        static partial void VerifyTryFormat(string test, string format, IFormatProvider provider, bool expectError, string expectedResult)
+        {
+            try
+            {
+                BigInteger bi = BigInteger.Parse(test, provider);
+
+                char[] destination = expectedResult != null ? new char[expectedResult.Length] : Array.Empty<char>();
+                bool success = bi.TryFormat(destination, out int charsWritten, format, provider);
+                Assert.False(expectError);
+                Assert.True(success);
+                Assert.Equal(expectedResult.Length, charsWritten);
+
+                string result = new string(destination, 0, charsWritten);
+                if (expectedResult != result)
```

Comment:
Rather than copied, can this be extracted to a reusable method in the other file?  (Because it's pretty strange...)

Reply:
All of these tests are fairly unmanageable.  Are you suggesting just the part from the if down to the end of the scope be separated out into a helper?  I can probably do that.

Reply:
Yep, that's what I was looking for.  Just that the body of that was hard to understand, so keeping it limited to one copy seemed good :)